registry.json

```diff
@@ -1741,5 +1741,30 @@
       "clientPublic": "https://rpc.ftm.tools",
       "clientDocs": "https://eth.wiki/json-rpc/API"
     }
+  },
+  {
+    "id": "cryptoorg",
+    "name": "Crypto.org",
```

Comment:
the generate type name is `TWCoinTypeCryptoorg`, can we do `TWCoinTypeCryptoOrg`? add display name to `Crypto.org`